src/Vertigo.Json/App.config

```diff
@@ -5,6 +5,6 @@
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.4.0.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.6.0.0" />
```

Comment:
does this have to be 4.6 (I havent even looked up what 4.6 means) ? (consider copying from jet.json.converters - not sure if that's more or less - but I guess if this will work)